.github/workflows/ci.yml

```diff
@@ -0,0 +1,41 @@
+name: Test
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby-version: ['2.3']
+        dist:
+          - ubuntu-14.04
+          - ubuntu-16.04
+          - debian-8
+          - debian-9
+          - centos-7
+          - amazonlinux
+          - amazonlinux-2
```

Comment:
Is there a reason why ubuntu-18.04 be disappearing?

Reply:
Oh, I just forgot it 😇